src/utils/fields.js

```diff
@@ -150,6 +151,7 @@ const FILE_VERSION_FIELDS_TO_FETCH = [
     FIELD_MODIFIED_BY,
     FIELD_RESTORED_FROM,
     FIELD_SIZE,
+    FIELD_PROMOTED_BY,
```

Comment:
🔤

Reply:
To be fair, it's not truly alphabetical lol